packages/webapp/pages/sources/[source].tsx

```diff
@@ -118,15 +120,13 @@ const SourcePage = ({ source }: SourcePageProps): ReactElement => {
         <span className="mr-auto">{source.name}</span>
         <Button
           className="laptop:hidden"
-          variant={ButtonVariant.Secondary}
           {...buttonProps}
           aria-label={unfollowingSource ? 'Follow' : 'Block'}
         />
-        <Button
-          className="hidden laptop:flex"
-          variant={ButtonVariant.Secondary}
-          {...buttonProps}
-        >
+        {!unfollowingSource && (
+          <SourceSubscribeButton className="ml-3 laptop:mr-3" source={source} />
+        )}
```

Comment:
Should this be hid behind a feature flag?

Reply:
Component is wrapped in [withExperiment](https://github.com/dailydotdev/apps/pull/2750#discussion_r1516128240) so it already is behind it.

Reply:
Ah yes! Didn't see that part 😬